xdebug_com.c

```diff
@@ -25,165 +25,205 @@
 # include <sys/select.h>
 #endif
 #ifndef PHP_WIN32
-#include <unistd.h>
-#endif
-
-#ifdef PHP_WIN32
-# include <process.h>
-# include <direct.h>
-# include "win32/time.h"
-#define PATH_MAX MAX_PATH
-#else
+# include <unistd.h>
 # include <sys/socket.h>
-# include <netinet/in.h>
 # include <netinet/tcp.h>
-# include <arpa/inet.h>
 # include <netdb.h>
+#else
+# include <process.h>
+# include <direct.h>
+# include "win32/time.h"
+# undef UNICODE
```

Comment:
The code was based on documentation from Microsoft. I believe that if you don't comment out UNICODE before including the rest of the headers there will be problems if the code does not work with the unicode versions of the functions. I have however not been able to test if this compiles on Windows, so there can be problems there.